pt_miniscreen/core/components/selectable_list.py

```diff
@@ -36,6 +36,14 @@ def selected_row(self):
     def is_empty(self):
         return len(self.rows) == 0
 
+    @property
+    def can_select_previous(self):
+        return 0 > self.state["selected_index"]
```

Comment:
this doesn't look right? 
```suggestion
        return self.state["selected_index"] > 0
```